sources/packages/api/src/types/config.ts

```diff
@@ -87,3 +88,11 @@ export interface CRAIntegrationConfig {
   programAreaCode: string;
   environmentCode: string;
 }
+
+export interface ATBCIntegrationConfig {
+  ATBCloginEndpoint: string;
```

Comment:
Please consider keeping the same pattern from other variables in the same object: "ATBCLoginEndpoint".